src/routes/accounts/entities/account-data-setting.entity.ts

```diff
@@ -0,0 +1,20 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+
+export class AccountDataSetting {
+  @ApiProperty()
+  dataTypeName: string;
+  @ApiPropertyOptional({ type: String, nullable: true })
+  dataTypeDescription: string | null;
```

Comment:
As above regarding the "dataType-" prefix.

Reply:
Changed in [7840a53](https://github.com/safe-global/safe-client-gateway/pull/1715/commits/7840a53a7fbd14e94d4b9428a7eaaec90bc7c712)